src/expr/src/vector_op/cast.rs

```diff
@@ -115,6 +117,17 @@ pub fn timestampz_to_utc_string(elem: i64) -> String {
     instant.format("%Y-%m-%d %H:%M:%S%.f%:z").to_string()
 }
 
+pub fn timestampz_to_utc_binary(elem: i64) -> Bytes {
+    // Just a meaningful representation as placeholder. The real implementation depends on TimeZone
+    // from session. See #3552.
+    let instant = Utc.timestamp_nanos(elem * 1000);
+    let mut out = BytesMut::new();
+    instant
+        .to_sql(&postgres_types::Type::ANY, &mut out)
```

Comment:
Why not `postgres_types::Type::TIMESTAMPTZ`?

Reply:
Did you say &postgres_types::Type::ANY? This is only a placeholder. I will add a comment later.

Reply:
Yes> I am curious why we pass `ANY` here

Reply:
This place needs a reference of postgres_types::Type, I just pick one postgres_types::Type casually 😂